doc/examples/mzml-mzqc-example.mzML

```diff
@@ -0,0 +1,448 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+<mzML xmlns="http://psi.hupo.org/ms/mzml" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://psi.hupo.org/ms/mzml http://psidev.info/files/ms/mzML/xsd/mzML1.1.0.xsd" accession="" version="1.1.0">
+	<cvList count="5">
+		<cv id="MS" fullName="Proteomics Standards Initiative Mass Spectrometry Ontology" URI="http://psidev.cvs.sourceforge.net/*checkout*/psidev/psi/psi-ms/mzML/controlledVocabulary/psi-ms.obo"/>
+		<cv id="UO" fullName="Unit Ontology" URI="http://obo.cvs.sourceforge.net/obo/obo/ontology/phenotype/unit.obo"/>
+		<cv id="BTO" fullName="BrendaTissue545" version="unknown" URI="http://www.brenda-enzymes.info/ontology/tissue/tree/update/update_files/BrendaTissueOBO"/>
+		<cv id="GO" fullName="Gene Ontology - Slim Versions" version="unknown" URI="http://www.geneontology.org/GO_slims/goslim_goa.obo"/>
+		<cv id="PATO" fullName="Quality ontology" version="unknown" URI="http://obo.cvs.sourceforge.net/*checkout*/obo/obo/ontology/phenotype/quality.obo"/>
+		<cv id="QC" fullName="Quality Control Ontology" version="unknown" URI="https://raw.githubusercontent.com/HUPO-PSI/mzQC/main/cv/qc-cv.obo"/>
+	</cvList>
+	<fileDescription>
+		<fileContent>
+			<cvParam cvRef="MS" accession="MS:1000294" name="mass spectrum" />
+		</fileContent>
+		<sourceFileList count="1">
+			<sourceFile id="sf_ru_0" name="BSA1_F1.mzML" location="file:///home/walzer/ms-tools/openms-source-git/share/OpenMS/examples/FRACTIONS">
+				<cvParam cvRef="MS" accession="MS:1000569" name="SHA-1" value="ce02b0744755a8d026f11ece3c5f9b76efcd87f8" />
+				<cvParam cvRef="MS" accession="MS:1000584" name="mzML format" />
+				<cvParam cvRef="MS" accession="MS:1000777" name="spectrum identifier nativeID format" />
+			</sourceFile>
+			<sourceFile id="QC1" name="BSA1_F1.mzQC" location="file:///">
```

Comment:
Is this file location expected to be empty?

Reply:
Was in the root of the container I built for creating the example. Did not include the filename, just the base path. The spec doc says about the location attribute:
> URI-formatted location where the file was retrieved.

I can see how this can be interpreted both base path and full path.
Maybe for our spec doc definition of locations, we should be more explicit? Right now we say for name and location:
>The name MUST uniquely match to a location (specified below) listed in the mzQC file.

And

>Unique file location, REQUIRED to be specified as a URI. The file URI is RECOMMENDED to be publicly accessible.
Should we require name 'redundancy'?

Reply:
I changed it to read `file:///examples/`

Reply:
Yes, my reading of the spec was always that `location` was supposed to be partially redundant with `name`. I've clarified it a bit in our spec name, check it out.